app/assets/stylesheets/css/fields/tags.css

```diff
@@ -22,10 +22,5 @@ tags.tagify {
 }
 
 tag.tagify__tag {
-  @apply text-sm !m-0;
-}
-
-/* When the tags field is used to select just one value there's a weird ZeroWidthSpace (\u200B) character that breaks the vertical spacing. */
```

Comment:
Why did you remove this bit @Paul-Bob?
Was it causing issues?

Reply:
Yes, when using in select mode that was a weird behavior, it's now on the input as `min-h-10`

Reply:
Ok. Thanks!